FBShimmering/FBShimmeringLayer.m

```diff
@@ -244,6 +244,22 @@ - (void)setShimmeringOpacity:(CGFloat)shimmeringOpacity
   }
 }
 
+- (CFTimeInterval)shimmeringBeginTime
+{
+  return [_maskLayer animationForKey:kFBShimmerSlideAnimationKey].beginTime;
+}
+
+- (void)resumeShimmeringWithBeginTime:(CFTimeInterval)beginTime
+{
+  CAAnimation *slideAnimation = [[_maskLayer animationForKey:kFBShimmerSlideAnimationKey] mutableCopy];
+  if (nil == slideAnimation) {
+    return;
+  }
+
+  slideAnimation.beginTime = beginTime;
+  [_maskLayer addAnimation:slideAnimation forKey:kFBShimmerSlideAnimationKey];
```

Comment:
But a `beginTime` of `0` could also be valid, no? For example, the `beginTime` of the fade animation is `0`.

Btw, instead of `CACurrentMediaTime()`, shouldn’t we be doing something like `[_maskLayer convertTime:CACurrentMediaTime() fromLayer:nil]`?